app/assets/javascripts/index/directions/fossgis_osrm.js

```diff
@@ -30,6 +30,7 @@
         "rotary": "roundabout",
         "exit roundabout": "exit_roundabout",
         "exit rotary": "exit_roundabout",
+        "ferry": "continue",
```

Comment:
Shouldn't this be:

```suggestion
        "ferry": "ferry",
```

As it stands I get the ferry icon but the text is still just "Continue on..." rather than the new ferry message.

Reply:
I shot myself in the foot by not testing that, didn't I?